pkg/kn/commands/domain/create.go

```diff
@@ -32,8 +33,8 @@ func NewDomainMappingCreateCommand(p *commands.KnParams) *cobra.Command {
 		Use:   "create NAME",
 		Short: "Create a domain mapping",
 		Example: `
-  # Create a domain mappings 'hello.example.com' for Knative service 'hello'
-  kn domain create hello.example.com --ref hello`,
+  # Create a domain mappings 'hello.example.com' for Knative service 'hello' with TLS secret set
+  kn domain create hello.example.com --ref hello --tls tls-secret`,
```

Comment:
Or maybe `tls-secret-name`, but let's present it like a user owned input.
```suggestion
  kn domain create hello.example.com --ref hello --tls my-tls-secret`,
```